packages/http/src/mocker/serializer/style/delimited.ts

```diff
@@ -0,0 +1,32 @@
+import { partial } from 'lodash';
+
+function serializeAndImplode(separator: string, name: string, value: Array<string | number | boolean>) {
```

Comment:
We actually do not care of the array type.
```suggestion
function serializeAndImplode(separator: string, name: string, value: Array<unknown>) {
```

Reply:
Why more permissive `unknown`?

Reply:
Actually is way _less_ permissive — unknown is basically a "we have no idea, so we won't make any assumption" and will prevent anybody inside the other functions to try to do some stuff (such as concatenate or add them)

Reply:
Yeah, but it will allow objects or arrays which we don't want.

Reply:
Oh gosh, you're right. You can ignore this, thanks for spotting it.